python/lsst/daf/butler/registries/sqlRegistry.py

```diff
@@ -21,7 +21,11 @@
 
 import itertools
 
+from sqlite3 import Connection as SQLite3Connection
```

Comment:
Any chance we could have the sqlite knowledge in a subclass?

Reply:
We probably should. But I'm not sure we want to do that right now?

Reply:
would it be anything other than copying 20 lines to another file and changing the registry name in the yaml file?